### PR TITLE
[DM-2164] Home Page rev2 doc update

### PR DESCRIPTION
**Pull request by dmajsag**
Update the user guide with  feature description and the relevant screenshots for Home Page rev2

### CODE REVIEW

File: content/users-guide/device-management-bundle/overview.md [hunk 1]
```diff
@@ -18,3 +18,15 @@ The following sections will walk you through all functionalities of the Device M
 |[Managing device data](#managing-device-data)|How to manage [firmware](#firmware-repo) and [software](#software-repo) for devices, [configuration snapshots](#configuration-repository), [device credentials](#credentials), [trusted certificates](#trusted-certificates) and [device profiles](#device-profiles).
 |[SmartREST templates](#smartrest-templates)|How to work with [SmartREST  templates](#smartrest-templates), a collection of request and response templates used to convert CSV data and {{< product-c8y-iot >}} Rest API calls.
 |[Working with simulators](#simulator)|How to model devices with the [simulator](#simulator) in order to have the same level of functionality as connected hardware devices.
+### Home dashboard
+
+The Home screen of the Device Management application is a dashboard which shows data for the general tenant.
```
Comment (rssag) [inline content/users-guide/device-management-bundle/overview.md:23]: what is the general tenant?

File: content/users-guide/device-management-bundle/overview.md [hunk 2]
```diff
@@ -18,3 +18,15 @@ The following sections will walk you through all functionalities of the Device M
...
+
+![Home dashboard](/images/users-guide/DeviceManagement/device-management-home-screen.png)
+
+The data shown on the Home dashboard is shared by all users of the tenant. By default, the Home dashboard includes recent alarms and quick links.
+The Home dashboard can be edited and designed individually according to your needs. You can add, remove or change widgets being displayed here.
```
Comment (rssag) [inline content/users-guide/device-management-bundle/overview.md:28]:
```suggestion
The Home dashboard can be edited and designed individually according to your needs. You can add, remove or change widgets which are displayed here.
```

File: content/users-guide/device-management-bundle/overview.md [hunk 3]
```diff
@@ -18,3 +18,15 @@ The following sections will walk you through all functionalities of the Device M
...
+
+For details on editing a dashboard, refer to [Working with dashboards](#dashboards).
+
+To reset the Home dashboard to its original content, click at the right of the top menu bar and then click **Restore dashboard**.
```
Comment (rssag) [inline content/users-guide/device-management-bundle/overview.md:32]:
```suggestion
To reset the Home dashboard to its default, click **Restore dashboard** at the right of the top menu bar. 
```

**Comment by rssag** [comment]
what are the target versions? @dmajsag

**Comment by dmajsag** [comment]
> what are the target versions? @dmajsag

10.17 is the target version

File: content/users-guide/device-management-bundle/overview.md (hunk 3 above)
Comment (dmajsag) [reply content/users-guide/device-management-bundle/overview.md]: Changed

File: content/users-guide/device-management-bundle/overview.md (hunk 2 above)
Comment (dmajsag) [reply content/users-guide/device-management-bundle/overview.md]: Changed

File: content/users-guide/device-management-bundle/overview.md (hunk 1 above)
Comment (dmajsag) [reply content/users-guide/device-management-bundle/overview.md]: The actual tenant ;) I have removed this

File: content/users-guide/device-management-bundle/overview.md [hunk 4]
```diff
@@ -18,3 +18,15 @@ The following sections will walk you through all functionalities of the Device M
...
+### Home dashboard
+
+The Home screen of the Device Management application is a dashboard which shows data for the general tenant.
+
+![Home dashboard](/images/users-guide/DeviceManagement/device-management-home-screen.png)
+
+The data shown on the Home dashboard is shared by all users of the tenant. By default, the Home dashboard includes recent alarms and quick links.
+The Home dashboard can be edited and designed individually according to your needs. You can add, remove or change widgets being displayed here.
+
+For details on editing a dashboard, refer to [Working with dashboards](#dashboards).
```
[the remark this answers is not in the document]
Comment (dmajsag) [reply content/users-guide/device-management-bundle/overview.md]: Commited suggestions

File: content/users-guide/device-management-bundle/overview.md [hunk 5]
```diff
@@ -18,3 +18,16 @@ The following sections will walk you through all functionalities of the Device M
 |[Managing device data](#managing-device-data)|How to manage [firmware](#firmware-repo) and [software](#software-repo) for devices, [configuration snapshots](#configuration-repository), [device credentials](#credentials), [trusted certificates](#trusted-certificates) and [device profiles](#device-profiles).
 |[SmartREST templates](#smartrest-templates)|How to work with [SmartREST  templates](#smartrest-templates), a collection of request and response templates used to convert CSV data and {{< product-c8y-iot >}} Rest API calls.
 |[Working with simulators](#simulator)|How to model devices with the [simulator](#simulator) in order to have the same level of functionality as connected hardware devices.
+### Home dashboard
```
Comment (rssag) [inline content/users-guide/device-management-bundle/overview.md:21]:
```suggestion

### Home dashboard
```

**Review by rssag — APPROVED** [review]

**Comment by rssag** [comment]
to merge it, the PR needs one more approval from your team @dmajsag

**Review by miryalarahul — APPROVED** [review]

**Pull request merged**